pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -258,8 +258,29 @@ func createMonitoringResources(engine chaosTypes.EngineInfo, recEngine *reconcil
 	return reconcile.Result{}, nil
 }
 
-// getChaosRunnerENV return the env required for chaos-runner
+//getChaosRunnerENV returns the env required for chaos-runner
 func getChaosRunnerENV(cr *litmuschaosv1alpha1.ChaosEngine, aExList []string, ClientUUID string) []corev1.EnvVar {
+	return append(getChaosRunnerDefaultENV(cr, aExList, ClientUUID), getResourceNamespace(os.Getenv("ADMIN_MODE")))
+}
+
+//getResourceNamespace returns the chaos resource namespace from adminMode
+func getResourceNamespace(adminMode string) corev1.EnvVar {
+
+	if adminMode == "true" {
+		return corev1.EnvVar{
+			Name:  "RESOURCE_NAMESPACE",
+			Value: os.Getenv("RESOURCE_NAMESPACE"),
```

Comment:
use `os.LookupEnv` so that you check for ENV is present or not something like:
```
resourceNamespace ,ok := os.LookupEnv("RESOURCE_NAMESPACE"),
```

Reply:
Rather than the error, we can check if the length of the string is > 0, will use os.GetEnv() to keep consistency throughout the code.